rskj-core/build.gradle

```diff
@@ -59,72 +58,72 @@ tasks.withType(AbstractArchiveTask) {
 
 ext {
     libVersions = [
-            nettyVer: '4.0.56.Final',
-            spongyCastleVer: '1.52.0.0',
-            bouncyCastleVer: '1.59',
-            leveldbVer: '0.9',
+            nettyVer               : '4.0.56.Final',
+            spongyCastleVer        : '1.52.0.0',
+            bouncyCastleVer        : '1.59',
+            leveldbVer             : '0.9',
             fusesourceLeveldbJniVer: '1.8',
-            ethereumLeveldbJniVer: '1.18.3',
-            slf4jVer: '1.7.25',
-            logbackVer: '1.2.2',
-            jacksonVer: '2.8.7',
-            commonsLang3Ver: '3.5',
-            typesafeConfigVer: '1.3.3',
-            mapdbVer: '2.0-beta13',
-            jsonrpc4jVer: '1.5.1',
-            jaxwsRtVer: '2.3.3',
-
-            bitcoinjThinVer: '0.14.4-rsk-10',
-            rskjNativeVer: '1.2.0',
+            ethereumLeveldbJniVer  : '1.18.3',
+            slf4jVer               : '1.7.25',
+            logbackVer             : '1.2.2',
+            jacksonVer             : '2.8.7',
+            commonsLang3Ver        : '3.5',
+            typesafeConfigVer      : '1.3.3',
+            mapdbVer               : '2.0-beta13',
+            jsonrpc4jVer           : '1.5.1',
+            jaxwsRtVer             : '2.3.3',
+
+            bitcoinjThinVer        : '0.14.4-rsk-10',
+            rskjNativeVer          : '1.2.0',
     ]
 
     libs = [
-            nettyLib: "io.netty:netty-codec-http:${libVersions.nettyVer}",
-            spongyCastleLib: "com.madgag.spongycastle:core:${libVersions.spongyCastleVer}",
-            bouncyCastleLib: "org.bouncycastle:bclcrypto-jdk15on:${libVersions.bouncyCastleVer}",
-            leveldbLib: "org.iq80.leveldb:leveldb:${libVersions.leveldbVer}",
+            nettyLib               : "io.netty:netty-codec-http:${libVersions.nettyVer}",
+            spongyCastleLib        : "com.madgag.spongycastle:core:${libVersions.spongyCastleVer}",
+            bouncyCastleLib        : "org.bouncycastle:bclcrypto-jdk15on:${libVersions.bouncyCastleVer}",
+            leveldbLib             : "org.iq80.leveldb:leveldb:${libVersions.leveldbVer}",
             fusesourceLeveldbJniLib: "org.fusesource.leveldbjni:leveldbjni:${libVersions.fusesourceLeveldbJniVer}",
```

Comment:
could you check if there's a need of `org.iq80.leveldb:leveldb` and `org.fusesource.leveldbjni:leveldbjni` libs in the project? It seems that they are not in use right now.

Reply:
Here for `iq80`: https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/org/ethereum/datasource/HashMapDB.java#L23 

Here for `fusesource`: https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/org/ethereum/datasource/LevelDbDataSource.java#L41

Reply:
do you think it could be that those two classes are also part of `leveldbjni-all-1.18.3.jar` ?

Reply:
It worked removing the these dependencies, already pushed.